oeps/processes/oep-0063-proc-toc-arbitration.rst

```diff
@@ -0,0 +1,115 @@
+.. _pep_based_template:
+
+.. Below is the display in the left sidebar on RTD. Please omit leading 0's
+
+OEP-63: TOC Arbitration
```

Comment:
@sarina My impression was that @e0d didn't like "TOC Decision", and I mentioned some objections to "Request for TOC Input" in https://github.com/openedx/open-edx-proposals/pull/484#issuecomment-1566819371 - so I was trying to find a good alternative. But I would be fine with "TOC Decision" - it has the advantage of being clear and simple.

Reply:
@kdmccormick agreed with "TOC Decision" here: https://github.com/openedx/open-edx-proposals/pull/484#issuecomment-1602763927 , and I didn't see Ed oppose it. I support TOC Decision as well. I'm not sure where "Arbitration" came from but in America, Arbitration is often a detestable process (ie in order to receive health care sometimes, you have to agree to arbitration in case something goes wrong, and other sort of ways corporations force you into arbitration)

Reply:
@sarina OK - since I'm also good with "TOC Decision", so I have made the change in https://github.com/openedx/open-edx-proposals/pull/484/commits/55d29d8b0cf87ad570c9f7fad6b0cc7ed29f611d :+1: